apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1470,10 +1474,23 @@ naming_system_manage_name(_Config) ->
     {ok, []} = rpc(aec_tx_pool, peek, [infinity]),
 
     %% Check that TTL and pointers got updated in name entry
-    TmpPointers = <<"{\"account_pubkey\":\"srtrst\"}">>,
     {ok, 200, #{<<"name">>     := Name,
                 <<"name_ttl">> := NameTTL,
-                <<"pointers">> := TmpPointers}} = get_name(Name),
+                <<"pointers">> := Pointers}} = get_name(Name),
+
+    {ok, 200, #{<<"balance">> := Balance3}} = get_balance(),
+    Host = internal_address(),
+    http_request(Host, post, "spend-tx",
+        #{recipient_pubkey => Name,
+          amount => 77,
+          fee => 50}),
+    aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE), 1),
+
+    %% Nothing gets lost as recipient = sender = miner
+    %% This tests 'resolve_name' because recipient is expressed by name label
+    %% This tests passes with 1 block confirmation due to lack of miner's reward delay
+    FinalBalance = Balance3+MineReward,
+    {ok, 200, #{<<"balance">> := FinalBalance}} = get_balance(),
```

Comment:
Test does not guarantee that spend tx included in block.
* Return code of spend tx HTTP call not checked;
* Mempool not checked after post spend tx;
* Balance post-mining is the same regardless spend tx was included or not.